release_notes.md

```diff
@@ -7,5 +7,5 @@
 
 **Release sprint:** Sprint 113
 [ [bugs](https://github.com/Azure/azure-functions-host/issues?q=is%3Aissue+milestone%3A%22Functions+Sprint+113%22+label%3Abug+is%3Aclosed) | [features](https://github.com/Azure/azure-functions-host/issues?q=is%3Aissue+milestone%3A%22Functions+Sprint+113%22+label%3Afeature+is%3Aclosed) ]
-- Update PowerShell Worker 7.0 to 4.0.1273 [Release Note](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v4.0.1273)
-- Update PowerShell Worker 7.2 to 4.0.1274 [Release Note](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v4.0.1274)
\ No newline at end of file
+- Update PowerShell Worker 7.0 to 4.0.1468 [Release Note](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v4.0.1468)
```

Comment:
Same question about the release notes as the other PR.

Reply:
@fabiocav -- If the release `4.1.3` was completed for `Sprint 113`, then the release notes were not reset. Otherwise, the PowerShell workers were updated twice since our last release.